apps/sasquatch/src/main/res/values/array.xml

```diff
@@ -12,4 +12,14 @@
         <item>NO_SECRET</item>
         <item>SKIP_START</item>
     </string-array>
+    <string-array name="appcenter_secret_values" tools:ignore="MissingTranslation">
+        <item>B2C</item>
+        <item>AAD</item>
+        <item>Custom</item>
+    </string-array>
+    <string-array name="appcenter_secret_guids" tools:ignore="MissingTranslation">
+        <item>9e0d97c1-7838-46d0-9dab-1a0ef66aec6e</item>
```

Comment:
This is the prod file, so prod values should be in this one.
There is another resource file for INT in projectDependency folder that needs different values for INT.

Reply:
Thanks, fixed.